pkgs/development/python-modules/cppe/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage, lib, cmake, cppe, eigen
+, python, pybind11, numpy, h5py, numba, scipy, pandas, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  inherit (cppe) pname version src;
+
+  # The python interface requires eigen3, but builds from a checkout in tree.
+  # Using the nixpkgs version instead.
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"external/eigen3",' '"${eigen}/include/eigen3",'
+  '';
+
+  nativeBuildInputs = [ cmake pytestCheckHook ];
```

Comment:
I'd change ``pytestCheckHook`` -> a ``checkInput``

Reply:
Not entirely sure what you mean. Do you mean putting `pytest` in `checkInput`? This one does not work with `disabledTestPaths` then.

Reply:
Nevermind, figured out what you've meant :smile: